### ISSUE TITLE
Socrata Dataset dominated by PDFs

### ISSUE THREAD

**Issue by allejo** [CityofSantaMonica/analytics.smgov.net#44]
With the current dataset, https://data.smgov.net/Public-Services/Web-Analytics/8dh4-6epx, I'm seeing that everything is being dominated by PDFs (should be related to #42). Here are some sample queries.

Everything's good so far,

```
https://data.smgov.net/resource/gi55-gz8u.json?&$where=date = '2016-08-01T00:00:00' AND domain = 'www.bigbluebus.com'
```

Starting on the 17th? No more pages are visible, only PDFs, so a backfill may be necessary once #42 is fixed and deployed.

```
https://data.smgov.net/resource/gi55-gz8u.json?&$where=date = '2016-08-17T00:00:00' AND domain = 'www.bigbluebus.com'
```

**Comment by thekaveman** [comment]
I don't think this is a real issue, but an artifact of the of the failing job from #42. 

For example, looking at the 16th (the last day the `socrata` job successfully ran), the following two queries will split the results:

**PDF requests**

```
https://data.smgov.net/resource/gi55-gz8u.json?&$where=date = '2016-08-16T00:00:00' AND domain = 'www.bigbluebus.com' AND page like '%25.pdf'
```

**non-PDF requests**

```
https://data.smgov.net/resource/gi55-gz8u.json?&$where=date = '2016-08-16T00:00:00' AND domain = 'www.bigbluebus.com' AND page not like '%25.pdf'
```

Once #42 is fixed, and we backfill for the missing days, things shouldn't look so weird.

**Comment by allejo** [comment]
For what it's worth, a report such as this one I mentioned here, https://github.com/CityofSantaMonica/analytics.smgov.net/issues/35#issuecomment-224200194, will probably work with some changes for back filling the missing data.

**Comment by thekaveman** [comment]
Not sure I follow... you mean for back-filling?

**Comment by allejo** [comment]
Yea, my bad for the lack of clarity

**Comment by thekaveman** [comment]
All good. I figured as much. 

Since this issue will be worked on and solved by #35 and #42, I'm going to close
